lambdas/service/app.py

```diff
@@ -1312,45 +1320,121 @@ def get_summary():
                     if initiate else
                     'Determine status of a manifest preparation job'
                 ) + (
                     ' via XHR' if fetch else ''
+                ) + (
+                    ' as an alternative to PUT for curl users' if curl else ''
                 ),
-            'description': fd('''
-                Create a manifest preparation job, returning either
+            'description': (
+                fd('''
+                    Create a manifest preparation job, returning either
 
-                - a 301 redirect to the URL of the status of that job or
+                    - a 301 redirect to the URL of the status of that job or
 
-                - a 302 redirect to the URL of an already prepared manifest.
-
-                This endpoint is not suitable for interactive use via the
-                Swagger UI. Please use [PUT /fetch/manifest/files][1] instead.
+                    - a 302 redirect to the URL of an already prepared manifest.
+                ''')
+                + iif(not curl, fd(f'''
+                    This endpoint is not suitable for interactive use via the
+                    Swagger UI. Please use [{method} /fetch/manifest/files][1]
+                    instead.
 
-                [1]: #operations-Manifests-put_fetch_manifest_files
-            ''') + parameter_hoisting_note('PUT', '/manifest/files', 'PUT')
+                    [1]: #operations-Manifests-{method.lower()}_fetch_manifest_files
+                '''))
+                + parameter_hoisting_note(method, '/manifest/files', method)
+                + iif(curl, fd(f'''
+                    Requests to this endpoint are idempotent, so PUT would be
+                    the more standards-compliant method to use. POST is offered
+                    as a convenience for `curl` users, exploiting the fact that
+                    `curl` drops to GET when following a redirect in response to
+                    a POST, but not a PUT request. This is the only reason for
+                    the deprecation of this endpoint and there are currently no
+                    plans to remove it.
+
+                    To use this endpoint with `curl`, pass the `--location` and
+                    `--data` options. This makes `curl` automatically follow the
+                    intermediate redirects to the GET /manifest/files endpoint,
+                    and ultimately to the URL that yields the manifest. Example:
+
+                    ```
+                    curl --data "" --location {post_manifest_example_url}
+                    ```
+
+                    In order to facilitate this, a POST request to this endpoint
+                    may have a `Content-Type` header of
+                    `application/x-www-form-urlencoded`, which is what the
+                    `--data` option sends. The body must be empty in that case
+                    and parameters cannot be hoisted as described above.
```

Comment:
This looks like something that was already there but it shows up as an addition without a corresponding deletion. IOW, it appears that this was copied and pasted.

Reply:
This originated from a [patch you provided](https://github.com/DataBiosphere/azul/pull/6099#pullrequestreview-2021163214), and was extended and refined [during review feedback](https://github.com/DataBiosphere/azul/pull/6099#discussion_r1596200378).